src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -351,6 +351,11 @@ export default {
 							placeholder: t('spreed', 'IRC channel'),
 							icon: 'icon-group',
 						},
+						channelPassword: {
```

Comment:
```suggestion
						channelpassword: {
```
all others are fully lowercase as well?